code/modules/mob/living/simple_animal/abnormality/aleph/nothing_there.dm

```diff
@@ -108,10 +111,20 @@
 	. = ..()
 	if(status_flags & GODMODE) // Contained
 		return
+	if((last_heal_time + 1 SECONDS) < world.time) // One Second between heals guaranteed
+		var/heal_amount = ((world.time - last_heal_time)/10)*heal_percent_per_second*maxHealth
+		if(health <= maxHealth*0.3)
+			heal_amount *= 2
+		adjustBruteLoss(-heal_amount)
+		last_heal_time = world.time
 	if(.)
 		if(next_transform && (world.time > next_transform))
 			next_stage()
 
+/mob/living/simple_animal/hostile/abnormality/nothing_there/apply_damage(damage, damagetype, def_zone, blocked, forced, spread_damage, wound_bonus, bare_wound_bonus, sharpness, white_healable)
+	. = ..()
+	last_heal_time = world.time + 5 SECONDS // Heal delayed when taking damage.
```

Comment:
As stated in the PR, I'm willing to move this number around a lot. Though this counts ALL sources of damage, so other abnos like Blue Star's ping or the Adoration can frequently reset this.